webcompat/static/js/lib/models/comment.js

```diff
@@ -25,5 +25,38 @@ issues.Comment = Backbone.Model.extend({
       createdAt: moment(response.created_at).fromNow(),
       rawBody: response.body
     });
+    if (this.parseHeader(jqXHR.xhr.getResponseHeader('Link')).last) {
+      response.lastPageNumber = this.parseHeader(jqXHR.xhr.getResponseHeader('Link')).last.split('\?page\=')[1];
+    }
+  },
+  parseHeader: function(linkHeader) {
```

Comment:
We should probably abstract this method out into a mixin, but we can do that in a follow up commit. Can you file a bug for that? And put a `//TODO: blah blah see Issue #XXXX.` comment as well?

That way we don't end up making changes in one of the models, but not the other.